src/build.py

```diff
@@ -940,17 +940,27 @@ def Wabt():
 
 def OCaml():
   buildbot.Step('OCaml')
+  # Build the ocaml compiler and runtime
   makefile = os.path.join(OCAML_DIR, 'config', 'Makefile')
   if not os.path.isfile(makefile):
     configure = os.path.join(OCAML_DIR, 'configure')
-    cc_flag = ['-cc', CC] if sys.platform != 'darwin' else []
+    cc_flag = ['-cc', CC,
```

Comment:
```python
cc_flag = # stuff
if sys.platform != 'darwin':
  cc_flag = []
```
?

Reply:
What, you don't like 2 line one-liners? :D